github/snapcraft.py

```diff
@@ -1,37 +1,126 @@
-#!/usr/bin/env python2
+#!/usr/bin/env/python2
 
-import urllib2
+import requests
 import urlparse
+import re
 
-
-def get_plugin_count(github_repo_url):
-    count = {}
-    repo = _get_repo_from_url(github_repo_url)
-    snapcraft_yaml_file = _get_snapcraft_yaml_file(repo)
-    if snapcraft_yaml_file:
-        for line in snapcraft_yaml_file:
-            line = line.strip()
-            if line.startswith('plugin: '):
-                plugin = line[8:]
-                if plugin in count:
-                    count[plugin] = count[plugin] + 1
-                else:
-                    count[plugin] = 1
-        snapcraft_yaml_file.close()
-    return count
+_SCRIPLETS = ["prepare", "build", "install", "version-script"]
+_tar_type_regex = re.compile(r'.*\.((tar(\.(xz|gz|bz2))?)|tgz)$')
 
 
 def _get_repo_from_url(github_repo_url):
+    """
+    Get full repository name from github url
+    :param github_repo_url: string, url
+    :return: string, full name
+    """
     parsed_url = urlparse.urlparse(github_repo_url)
     return parsed_url.path[1:]
 
 
-def _get_snapcraft_yaml_file(github_repo):
+def get_snapcraft_yaml_file(github_repo_url):
+    """
+    Extracts snapcraft file from repository
+    :param github_repo_url: string, url of the repository
+    :return: string, text of yaml file
+    """
+
+    github_repo = _get_repo_from_url(github_repo_url)
     paths = ['snapcraft.yaml', '.snapcraft.yaml', 'snap/snapcraft.yaml']
     for path in paths:
         url = 'https://raw.githubusercontent.com/{}/master/{}'.format(
             github_repo, path)
-        try:
-            return urllib2.urlopen(url)
-        except:
-            pass
+        response = requests.request("GET", url)
+
+        if response.status_code == 200:
+            return response.text
+
```

Comment:
The spacing should be consistent. Top-level functions should be separated by two empty  lines.